vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/registry/impl/InstallationScope.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jackrabbit.vault.packaging.registry.impl;
+
+public final class InstallationScope {
```

Comment:
wouldn't a `enum` be better ?

Reply:
@tripodsan  - actually I started with it but the problem is that there is no static access to the string values of an enum making this unusable for the configuration case - I'll switch that to a normal POJO with constants and getters/setters for a single value so I can use it in the internals to pass not string objects.  Patched now

Reply:
`InstallationScope.UNSCOPED.name()` ?

Reply:
.name() is also not a constant expression (as it may be overridden) - getting corresponding errors.